### PR TITLE
Quinn nilsson

### PR DESCRIPTION
**Pull request by QNilsson**
trying to merge quinnNilsson into master

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import SignIn from './views/SignIn';
 import Yahtzee from './components/Yahtzee';
 import Nertz from './views/games/Nertz';
 import YuGiOh from './views/YuGiOh';
+import Qwixx from './components/Qwixx';
```
Comment (skaduusch) [inline src/routes.js]: We should move your Qwixx component to the /views/games folder instead of /components

Comment (QNilsson) [reply src/routes.js]: Yeah I tried to do that yesterday but then nothing would load on my Qwixx page when i tested it so I put it back in components so I could see what I was editing.

File: src/App.vue [hunk 2]
```diff
@@ -1,87 +1,56 @@
 <template>
 	<v-app id="inspire">
-		<v-navigation-drawer
-			v-model="drawer"
-			:clipped="$vuetify.breakpoint.lgAndUp"
-			app
-			hide-default-footer
```
Comment (skaduusch) [inline src/App.vue]: Was this removed on purpose?

File: src/App.vue [hunk 3]
```diff
@@ -1,87 +1,56 @@
...
-				</v-list-item-content>
-			</v-list-item>
+		<v-navigation-drawer v-model="drawer" :clipped="$vuetify.breakpoint.lgAndUp" app>
 			<v-list dense>
 				<template v-for="item in items">
-					<v-row
-						v-if="item.heading"
-						:key="item.heading"
-						align="center"
-						:to="item.to"
```
Comment (skaduusch) [inline src/App.vue]: Was this removed on purpose?

File: src/App.vue [hunk 4]
```diff
@@ -1,87 +1,56 @@
...
 						</template>
-						<v-list-item
-							v-for="(child, i) in item.children"
-							:key="i"
-							:to="child.to"
-							link
-						>
+						<v-list-item v-for="(child, i) in item.children" :key="i" link :to="child.to">
 							<v-list-item-action v-if="child.icon">
-								<v-icon :color="child.color">{{ child.icon }}</v-icon>
```
Comment (skaduusch) [inline src/App.vue]: Was this color option removed on purpose? :color="child.color"

File: src/App.vue [hunk 5]
```diff
@@ -1,87 +1,56 @@
...
-								<v-list-item-title>
-									{{ child.text }}
-								</v-list-item-title>
+								<v-list-item-title>{{ child.text }}</v-list-item-title>
 							</v-list-item-content>
 						</v-list-item>
 					</v-list-group>
 					<v-list-item v-else :key="item.text" link>
 						<v-list-item-action>
-							<v-icon :color="item.color">{{ item.icon }}</v-icon>
```
Comment (skaduusch) [inline src/App.vue]: Same as above

File: src/routes.js (hunk 1 above)
Comment (skaduusch) [reply src/routes.js]: You should just be able to move it and then update the path on this line to match.

Comment (QNilsson) [reply src/routes.js]: hmm yeah i tried that exact path and it wasnt working but i'll try it again

File: src/App.vue (hunk 2 above)
Comment (QNilsson) [reply src/App.vue]: No it was not removed on purpose, I dont know how all these changes in App.vue happened I didnt modify the code except for adding some paths to my qwixx component..

File: src/App.vue [hunk 6]
```diff
@@ -191,7 +189,8 @@ export default {
 						text: 'Quix',
 						icon: 'mdi-dice-5-outline',
 						color: 'green',
-						to: '/quix' },
+						to: '/qwixx' ,
```
Comment (QNilsson) [inline src/App.vue:192]: just did a name correction. that is the case for most of these lines

File: src/routes.js [hunk 7]
```diff
@@ -16,4 +17,5 @@ export const routes = [
 	{ path: '/nertz/:gameId', component: Nertz },
 	{ path: '/yugioh', component: YuGiOh },
 	{ path: '/ohhell', component: OhHell },
+	{ path: '/qwixx', component: Qwixx },
```
Comment (QNilsson) [inline src/routes.js:20]: added path to my game

File: src/views/Home.vue [hunk 8]
```diff
@@ -44,7 +43,7 @@ export default {
 			{
 				text: 'Qwixx',
 				img: require('../assets/images/quix.jpg'),
-				to: 'qwuixx',
+				to: '/qwixx',
```
Comment (QNilsson) [inline src/views/Home.vue:46]: name change.

File: src/views/games/Qwixx.vue [hunk 9]
```diff
@@ -0,0 +1,205 @@
...
+			<h2 class="display-3 font-weight-black underlined">Qwixx</h2>
+		</v-row>
+		<v-row>
+			<input class="mx-4 my-4" v-model="name" placeholder="Put name here" />
+			<v-btn small color="white">Submit</v-btn>
+		</v-row>
+
+		
+		<v-row class="mb-6">
+			<div class="redBtns mr-3 mb-2" v-for="redBtn in redBtns" :key="redBtn.id">
```
Comment (CobyYates) [inline src/views/games/Qwixx.vue:13]: I pulled your code and saw errors with key values. I would just change the v-for's to this
v-for="(redBtn, i) in redBtns" :key="i" for all four sections of buttons.

**Comment by QNilsson** [comment]
oh i didnt see that, thanks i'll change it

File: src/views/games/Qwixx.vue [hunk 10]
```diff
@@ -0,0 +1,205 @@
...
+					dark
+					@click="subtract(pen.v, pen.e = true)"
+					:disabled="pen.e"
+					v-for="(pen, i) in penalties"
+					:key="i"
+				>{{ pen.v }}</v-btn>
+			</v-col>
+		</v-row>
+
+		<v-row class="mb-6" justify="flex-start" no-gutters>
```
Comment (CobyYates) [inline src/views/games/Qwixx.vue:59]: I would also change line 59 to <v-row class="mb-6 d-flex justify-start" no-gutters> the justify="flex-start" was throwing an error as well.

**Pull request merged**